container-lite/pom.xml

```diff
@@ -223,7 +223,6 @@
               <repository>docker.io/${docker-organization}/job-lite</repository>
               <tag>${docker-tag}</tag>
               <executable>${basedir}/container-push.sh</executable>
-              <!-- DOCKER_USERNAME and DOCKER_PASSWORD must be set externally. -->
```

Comment:
Why remove this comment ?

Reply:
Probably removed by mistake when adjusting this file for local build. 
Restored.